rest-server/engines/src/main/scala/com/actionml/engines/ur/UREngine.scala

```diff
@@ -235,6 +235,29 @@ object UREngine extends JsonSupport {
       @SingleIndex(order = desc, isTtl = true) eventTime: Date)
     extends Event with Serializable with Response
 
+  case class UREventOriginal(
+    eventId: Option[String],
+    event: String,
+    entityType: String,
+    entityId: String,
+    targetEntityId: Option[String],
+    targetEntityType: String = "item",
+    properties: Map[String, Any],
+    eventTime: Date
+  ) extends Response
+  object UREventOriginal {
+    def fromUREvent(e: UREvent): UREventOriginal =
+      UREventOriginal(
+        e.eventId,
+        e.event,
+        e.entityType,
+        e.entityId,
+        e.targetEntityId,
+        properties = e.dateProps ++ e.floatProps ++ e.booleanProps ++ e.categoricalProps,
```

Comment:
have you tried this with actual properties? I'm unsure if this creates the correct format.

In our case we should never have properties so I'll approve but this should at least be commented as untested.

Reply:
yes, I tried, all looks good - {}